modules/microshift-ingress-controller-config.adoc

```diff
@@ -0,0 +1,153 @@
+* `Ignore`, adds the `http-ignore-probes` parameter in the `HAproxy configuration. If the field is set to `Ignore`, the ingress controller closes the connection without sending a response, then logs the connection or incrementing metrics.
+
+Usually, empty request connections come from load balancer health probes or web browser preconnects and can be safely ignored. However, network errors and port scans can also create these empty requests, so setting this field to `Ignore` can impede detecting or diagnosing problems and impede detecting intrusion attempts.
+
+|`logEmptyRequests`
+|`logEmptyRequests` specifies connections for which no request is received and logged. Usually, these empty requests come from load balancer health probes or web browser speculative connections (preconnect) and logging these requests can be undesirable. However, network errors and port scans can also create these empty requests, so setting this field to `Ignore` can impede detecting or diagnosing problems and impede detecting intrusion attempts.
+
+The following are valid values:
+
+* `Log`, indicates that an event should be logged.
+* `Ignore`, sets the `dontlognull` option in the `HAproxy` configuration.
+
+|`tuningOptions`
+|`tuningOptions` specifies options for tuning the performance of ingress controller pods.
+
+* `tuningOptions.clientFinTimeout` specifies how long a connection is held open while waiting for the client response to the server closing the connection. The default timeout is `1s`.
+
+* `tuningOptions.clientTimeout` specifies how long a connection is held open while waiting for a client response. The default timeout is `30s`.
+
+* `tuningOptions.headerBufferBytes` specifies how much memory is reserved, in bytes, for Ingress Controller connection sessions. This value must be at least `16384` if HTTP/2 is enabled for the Ingress Controller. If not set, the default value is `32768` bytes.
++
+[IMPORTANT]
+====
+Setting this field not recommended because `headerBufferMaxRewriteBytes` values that are too small can break the ingress controller. Conversely, values for `headerBufferMaxRewriteBytes` that are too large could cause the ingress controller to use significantly more memory than necessary.
+====
+
+* `tuningOptions.headerBufferMaxRewriteBytes` specifies how much memory should be reserved, in bytes, from `headerBufferBytes` for HTTP header rewriting and appending for Ingress Controller connection sessions. The minimum value for `headerBufferMaxRewriteBytes` is `4096`. `headerBufferBytes` must be greater than `headerBufferMaxRewriteBytes` for incoming HTTP requests. If not set, the default value is `8192` bytes.
++
+[IMPORTANT]
+====
+Setting this field not recommended because `headerBufferMaxRewriteBytes` values that are too small can break the ingress controller. Conversely, values for `headerBufferMaxRewriteBytes` that are too large could cause the ingress controller to use significantly more memory than necessary.
```

Comment:
Same question about capitalization for this admonition and the next one in this table cell